### ISSUE TITLE
Tradingview chart snapshots included in webhook message! 

### ISSUE THREAD

**Issue by mxwo93** [Drakkar-Software/OctoBot#1725]
Hi!

Is there ANY way to have a chart with the corresponding timeframe sent along with the webhook message to Discord?
Potentially using Puppeteer?

Maybe something like [THIS](https://github.com/isaacnass/discord-puppeteer-bot)?

I'm very new to coding and am REALLY trying to implement chart snapshots into the webhook message so people can visualize where the pair is at.

If ANYONE can help me at all, this has been killing me for the entire week.

THANK YOU!

**Comment by GuillaumeDSM** [comment]
Hi,
It looks like the `{{interval}}` value is proving the timeframe data in trading view webhook messages:
![image](https://user-images.githubusercontent.com/9078616/120441913-5d668a80-c385-11eb-90a1-3ca131f57071.png)

**Comment by mxwo93** [comment]
> Hi,
> It looks like the `{{interval}}` value is proving the timeframe data in trading view webhook messages:
> ![image](https://user-images.githubusercontent.com/9078616/120441913-5d668a80-c385-11eb-90a1-3ca131f57071.png)

Yes, that's true, but how can I then create and have a graph posted along with the message? I'm very new to coding so any help is greatly appreciated.

Going to try to get the basic Discord webhook set up now.

**Comment by GuillaumeDSM** [comment]
Maybe the `{{plot_}}` variables can help, not sure If it's really possible to send an image though, this requires to dig deeper in tradingview. Maybe you can create an image from datasets in `{{plot_}}`.

**Comment by mxwo93** [comment]
> Maybe the `{{plot_}}` variables can help, not sure If it's really possible to send an image though, this requires to dig deeper in tradingview. Maybe you can create an image from datasets in `{{plot_}}`.

What about using something like Puppeteer or Selenium to pull an image of the chart URL or snapshot URL?

**Comment by GuillaumeDSM** [comment]
It might work but will be really complicated, the OctoBot price chart is not enough for displaying the current price ? I'm trying to understand what you want to achieve exactly

**Comment by mxwo93** [comment]
> It might work but will be really complicated, the OctoBot price chart is not enough for displaying the current price ? I'm trying to understand what you want to achieve exactly

THIS:
https://github.com/itsaLca/Advanced-Tradingview-Webhook

Supposed to do exactly what I'd like, but it doesn't. Send a chart along with the notification.

**Comment by mxwo93** [comment]
Unable to add Discord as a service anyways, so kind of pointless. Oh well!

**Comment by GuillaumeDSM** [comment]
Telegram is supported but not Discord yet, we will add it in the future

**Comment by mxwo93** [comment]
Understood! I'll route the messages from Telegram to Discord.. However, my Webhook URL isn't showing up so I'm unable to have alerts sent to OctoBot? I've added Tradingview to my interfaces/connections. I am using the Tradingview alert strategy. For some reason no webhook URL shows up next to the token on the interfaces/connections page next to Tradingview or Web or Webhook...

How do I get my Webhook URL?

How do I fix this?
